rust/cloud-storage/authentication_service/src/api/user/post_get_names_with_email.rs

```diff
@@ -0,0 +1,44 @@
+use axum::{
+    Extension, Json,
+    extract::{self, State},
+    http::StatusCode,
+    response::{IntoResponse, Response},
+};
+use macro_db_client::user::get_user_name::get_user_names_with_email;
+
+use crate::api::context::ApiContext;
+
+use model::response::ErrorResponse;
+use model::user::{UserContext, UserNames};
+
+#[derive(Default, Debug, serde::Serialize, serde::Deserialize, utoipa::ToSchema)]
+pub struct GetNamesWithEmailRequestBody {
```

Comment:
This is a duplicate of the get_names request body, can you just use that here as well.